app/views/ncr/fields/_rwa_number.html.haml

```diff
@@ -1,4 +1,4 @@
-.column{ class: key + "-wrapper", id: key + '-wrapper-3', data: {"filter-key": "expense-type", "filter-value": "BA80"}}
+.column{ class: key + "-wrapper half-col-wrap sub-card sub-card-left", id: key + '-wrapper-3', data: {"filter-key": "expense-type", "filter-value": "BA80"}}
```

Comment:
New classes to manage the "sub card" design, instead of manually using the classes on the specific fields